packages/kbn-optimizer/src/worker/run_compilers.ts

```diff
@@ -182,7 +182,7 @@ const observeCompiler = (
       );
 
       bundle.cache.set({
-        bundleRefExportIds,
+        bundleRefExportIds: bundleRefExportIds.sort((a, b) => a.localeCompare(b)),
```

Comment:
note: we already sort `files` before calculating cache key https://github.com/elastic/kibana/pull/80011/files#diff-8d312ccddb403471ce12382f3e85d001R169

Reply:
Would you mind using the sort helpers for consistency? `ascending((id) => id)`